src/App.tsx

```diff
@@ -72,7 +72,8 @@ const useStyles = makeStyles((theme: Theme) => ({
     height: '100%',
     overflow: 'auto',
     '& > div, & > main': {
-      paddingTop: '64px',
+      paddingBottom: theme.spacing(25),
```

Comment:
Thanks for the fix @joshbuchea - this fixes the accessions create scroll issue, tested on vercel.
Do we want this on all screens? It is harmless ofc except for extra scrolling on all pages, happens on empty state content as well.

Reply:
Fair enough, we can revisit if design has concerns.